README.md

```diff
@@ -181,6 +181,23 @@ Class names like `Options` and `Level` might be too common, if you get a conflic
     })
     ...
 
+#### Angular Universal
+
+To use it with Angular Universal add angular2-logger to App.module normaly and in node.module provide these custom options.   
+
+```javascript
+import { Options } from "angular2-logger/core";
+
+@NgModule({
+  bootstrap: [ AppComponent ],
+  imports: [ AppModule, ... ],
+  providers: [
+    { provide: Options, useValue: { global: false, store: false } }, // stop angular2-logger from defining global variables.
```

Comment:
@FelipeTaiarol There's a confusion between two different things, one is who "generates" the HTML code and a different one is who executes it / renders it (who renders the page, not who parses the HTML code). This change can't be applied as is, as you'll be breaking the library functionality for those using it in the browser with the change in logger.ts